theano/gof/unify.py

```diff
@@ -229,7 +226,7 @@ def unify_walk(a, b, U):
         return False
 
 
-@comm_guard(FreeVariable, ANY_TYPE)
+@comm_guard(FreeVariable, ANY_TYPE)  # noqa
```

Comment:
no run travis, but you can generete the doc with:

python doc/scripts/docgen.py

it should have no warning.

and

nosetests theano/tests/test_flake8.py

On Mon, Jun 13, 2016 at 10:03 AM, Chinna <notifications@github.com> wrote:

> In theano/gof/unify.py
> <https://github.com/Theano/Theano/pull/4616#discussion_r66796244>:
>
> > @@ -229,7 +226,7 @@ def unify_walk(a, b, U):
> >          return False
> >
> >
> > -@comm_guard(FreeVariable, ANY_TYPE)
> > +@comm_guard(FreeVariable, ANY_TYPE)  # noqa
>
> Mathieu suggested to add # noqa to avoid errors like "
> /u/sankarch/Documents/theano_dev/theano/gof/unify.py:230:1: F811
> redefinition of unused 'unify_walk' from line 192" in unify.py. I am
> correcting the others.Is there a way to run travis by myself? Thanks Chinna
> … <#m_-3610053007879351648_>
> On Mon, Jun 13, 2016 at 10:00 AM, Frédéric Bastien ***@***.*** > wrote: In
> theano/gof/unify.py <#4616 (comment)
> <https://github.com/Theano/Theano/pull/4616#discussion_r66795734>>: > @@
> -229,7 +226,7 @@ def unify_walk(a, b, U): > return False > > >
> ***@***.***_guard(FreeVariable, ANY_TYPE) > ***@***.***_guard(FreeVariable,
> ANY_TYPE) # noqa why noqa? — You are receiving this because you authored
> the thread. Reply to this email directly, view it on GitHub <
> https://github.com/Theano/Theano/pull/4616/files/903dddedaec4bc5079863415c45f9d7680eee743#r66795734>,
> or mute the thread <
> https://github.com/notifications/unsubscribe/AJD2q82BujuFp3uQYFR2oECJfzyvkzn7ks5qLWJpgaJpZM4IzdlP>
> .
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/Theano/Theano/pull/4616/files/903dddedaec4bc5079863415c45f9d7680eee743#r66796244>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/AALC-7c6JzBHBpK9AG1xZ2NYzc4d3dwbks5qLWMUgaJpZM4IzdlP>
> .
>